framework/src/main/java/org/apache/drill/test/framework/TestDriver.java

```diff
@@ -215,6 +212,14 @@ public void generateReports(List<DrillTest> tests, int iteration) {
       }
       bufferedWriter.flush();
       bufferedWriter.close();
+
+      // Upload report to DFS if the drillReportsDFSDir variable is set
+      if (!drillReportsDFSDir.isEmpty()){
```

Comment:
This is done. The local report file is removed automatically after it's copied into DFS.